java/src/main/java/org/extism/sdk/manifest/Manifest.java

```diff
@@ -0,0 +1,64 @@
+package org.extism.sdk.manifest;
+
+import org.extism.sdk.wasm.WasmSource;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class Manifest {
+
+    private final List<WasmSource> wasm;
+
+    private final ManifestMemory memory;
```

Comment:
Memory and allowedHosts is currently not used? We should either use this OR remove it for now.